internal/otelcollector/config/config_base_processors.go

```diff
@@ -1,4 +1,4 @@
-package common
+package config
 
 type BaseProcessorsConfig struct {
```

Comment:
I also would prefer this type to be BaseProcessor. (`config.BaseProcessor` vs `config.BaseProcessorConfig`).

So the idea is to assume that all the types inside the `config` package are configs and to strip the Config suffix for them. What do you think?

Reply:
Yeah, I like the idea